Netable/Example/Views/CreateGraphQLPostView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CreateGraphQLPostView.swift
+//  Example
+//
+//  Created by Amy Oulton on 2022-12-08.
+//  Copyright © 2022 Steamclock Software. All rights reserved.
+//
+
+import SwiftUI
+
+struct CreateGraphQLPostView: View {
+    @Environment(\.presentationMode) var presentationMode
+    @ObservedObject var viewModel: GraphQLVM
+
```

Comment:
I got a little carried away with the SwiftUI here but the form was basically invisible without it 🙃